contracts/Registry.sol

```diff
@@ -204,14 +213,15 @@ contract Registry is IRegistry, ERC165 {
             }
 
             // shorten keys array
-            orgsById[orgId].memberKeys.length--;
+            //orgsById[orgId].memberKeys.length--;
```

Comment:
It is probably cleaner to remove old commented code completely (there are several "length--" comments here)

Reply:
Thanks, kept it for internal reference. Removed the commented code.